test/test_functional.py

```diff
@@ -0,0 +1,182 @@
+import math
+import os
+
+import torch
+import torchaudio
+import unittest
+import test.common_utils
+
+
+class TestFunctional(unittest.TestCase):
+    data_sizes = (2, 20)
```

Comment:
I'd add multiple here, this won't help you find various boundary cases.

Reply:
Done.